contributingGuides/CONTRIBUTING.md

```diff
@@ -49,7 +49,7 @@ If you've found a vulnerability, please email security@expensify.com with the su
 ## Payment for Contributions
 We hire and pay external contributors via Upwork.com. If you'd like to be paid for contributing or reporting a bug, please create an Upwork account, apply for an available job in [GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22), and finally apply for the job in Upwork once your proposal gets selected in GitHub. If you think your compensation should be increased for a specific job, you can request a reevaluation by commenting in the Github issue where the Upwork job was posted. 
 
-Payment for your contributions and bug reports will be made no less than 7 days after the pull request is deployed to production to allow for regression testing. If a regression occurs, payment will be issued 7 days after all regressions are fixed. If you have not received payment after 8 days of the PR being deployed to production and there being no regressions, please email contributors@expensify.com referencing the GH issue and your GH handle. 
+Payment for your contributions and bug reports will be made no less than 7 days after the pull request is deployed to production to allow for regression testing. If a regression occurs, payment will be issued 7 days after all regressions are fixed (ie: deployed to production). A 50% penalty will be applied to the Contributor and Contributor+ for each regression on an issue. If you have not received payment after 8 days of the PR being deployed to production and there being no regressions, please email contributors@expensify.com referencing the GH issue and your GH handle. 
```

Comment:
```suggestion
Payment for your contributions and bug reports will be made no less than 7 days after the pull request is deployed to production to allow for regression testing. If a regression occurs, payment will be issued 7 days after all regressions are fixed (ie: deployed to production). A 50% penalty will be applied to the Contributor and Contributor+ for each regression on an issue. If you have not received payment after 8 days of the PR being deployed to production, and there are no regressions, please email contributors@expensify.com referencing the GH issue and your GH handle. 
```

Reply:
additionally, with `A 50% penalty will be applied to the Contributor ` is this to the **total** amount? Say a bug bounty was 1000, they achieved the timeline bonus of 50% so it's 1500, BUT they introduced the regression, the 50% would be deducted from the $1500, right?

Reply:
@sophiepintoraetz 
> If the PR causes a regression at any point within the regression period (starting when the code is merged and ending 7 days after being deployed to production), contributors are not eligible for the 50% bonus.

The 50% would be deducted from the $1000.

Reply:
I'm not sure I agree with that, but if we do decide to go down that route, then I think we need to make that clearer (e.g. 50% will be deducted from the bug bounty total)

Reply:
It won't make sense to deduct from $1500 because the issue won't be eligible for the bonus in such a case.

Reply:
That's not true though, right? The current process is:
1. PR is merged within 3 days = eligible for bonus (i.e. the total amount for the bug is already established)
2. PR is pushed to production
3. 7 day regression period applied
4. Payment based on 7 day regression

Reply:
AFAIK If a PR caused a regression it loses that bonus privilege.

Reply:
I agree with you but I don't think it's been agreed upon widely. So if we're saying that's the case, I think we need to call that out more explicitly in this message (@conorpendergrast - do you agree?)